client/session.go

```diff
@@ -418,7 +417,11 @@ func (s *session) FetchAll(ids []string, startInclusive, endExclusive time.Time)
 				iterIdx := atomic.AddInt32(&success, 1) - 1
 				results[iterIdx] = multiIter
 			}
-			if remaining != 0 {
+			// NB(xichen): decrementing pending and checking remaining against zero must
```

Comment:
Maybe merge master so this doesn't show up as a change?  I forget the best way to do that on GitHub =[